Document changes to unknown insert tags (#4934)
contao/contao: UPGRADE.md

## Version 4.* to 5.0

### Unknown insert tags

> Unknown insert tags are no longer removed from the resulting text. Instead they are now kept as is and are visible in
> the front end.

Reviewer: ```suggestion
Unknown insert tags are no longer removed from the resulting text. Instead, they are now kept unchanged and are visible
in the front end.
```
